### DISCUSSION TITLE
Typo in the code?

### DISCUSSION THREAD

**Discussion by skylarl07** [python-gitlab/python-gitlab#1857] [Q&A] [answered]
I found line 362 in `gitlab.base`. I thought this would be `if path is not None:`

[https://github.com/python-gitlab/python-gitlab/blob/ff4b1cc70910b5c45df96547815390736a550b54/gitlab/base.py#L362](url)

**Comment by nejch** [comment, marked as answer]
Hi @skylarl07, I don't think so. The code is saying "if no `path` is supplied as an argument to the method, use the class instance's own `self._path`. In the next if block, if the `path` is _still_ `None`, then return `None`.

Either way, that line has been in the codebase for 5 years with plenty of testing and use in the wild, so should be fine, unless you had some specific concerns there. I hope that answers your question, but let us know otherwise!

**Reply by skylarl07** [reply]
Thank you that makes sense!
